staging/src/k8s.io/client-go/examples/third-party-resources/client.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/rest"
+)
+
+func NewClient(cfg *rest.Config) (*rest.RESTClient, *runtime.Scheme, error) {
+	groupVersion := schema.GroupVersion{
+		Group:   ExampleResourceGroup,
+		Version: ExampleResourceVersion,
+	}
+
+	schemeBuilder := runtime.NewSchemeBuilder(func(scheme *runtime.Scheme) error {
+		scheme.AddKnownTypes(
+			groupVersion,
+			&Example{},
+			&ExampleList{},
+		)
```

Comment:
`&metav1.ListOptions{}` and `&metav1.DeleteOptions{}` should be added here otherwise get/list and delete will not work if this scheme object is used with client.